test/endtoend/utils.sh

```diff
@@ -103,17 +103,22 @@ function takeBackup() {
 
   sleep 10
 
+  mysql 'commerce:-' -e "flush binary logs" || echo "flushing binlogs failed"
```

Comment:
The incremental backup will `flush binary logs` and there is no need to to that manually. If anything, it's better to _not_ `flush binary logs` so that the backup's behavior is better tested.